server/src-lib/Hasura/RQL/DDL/Metadata/Generator.hs

```diff
@@ -345,3 +406,19 @@ sampleCronSchedules = rights $ map Cr.parseCronSchedule $
   , "0 0 * * 0"
   -- Every sunday at 00:00
   ]
+
```

Comment:
Explain why these values are hardcoded here, (due to no `Arbitrary` instance for `Void`)